### PR TITLE
added blog post about micro-ROS at ERF 2021

### PR DESCRIPTION
**Pull request by JanStaschulat**
tested links to pdfs and external links locally with `bundle exec jekyll serve`.

### CODE REVIEW

**Comment by JanStaschulat** [comment]
output of the htlm checker:
`
Checking 779 external links...
Ran on 141 files!
- ./_site/docs/concepts/rtos/comparison/index.html
  *  External link https://interactive.freertos.org/hc/en-us/community/posts/210029046-POSIX-Wrapper-for-FreeRTOS failed: 403 No error
/home/runner/work/micro-ROS.github.io/micro-ROS.github.io/vendor/bundle/ruby/2.7.0/gems/html-proofer-3.15.1/lib/html-proofer/runner.rb:172:in `print_failed_tests': HTML-Proofer found 1 failure! (RuntimeError)
`

but the link `https://interactive.freertos.org/hc/en-us/community/posts/210029046-POSIX-Wrapper-for-FreeRTOS` is correct. 
@pablogs9 any idea?

File: _posts/2021-04-13-ERF2021.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: micro-ROS at European Robotics Forum 2021
+author: JanStaschulat
+---
+
+On April 13th 2021, we presented micro-ROS at the [European Robotics Forum 2021](https://www.eu-robotics.net/robotics_forum/) 
+
+<img alt="Deeply Embedded Software Workshop" src="/img/posts/2021-04-13-micro-ROS-at-ERF2021.png" width="80%"/>
+
+In this workshop, Francesca Finoccharo (eProsima) presented in detail the micro-ROS architecture, supported platforms and reported recent developments and features: A versatile API for custom transports, continious fragment mode, static memory pools in the RMW and time synchronization beween client and agent. ([slides](/download/2021-04-13-ERF-eprosima.pdf))
```
Comment (pablogs9) [inline _posts/2021-04-13-ERF2021.md:10]:
```suggestion
In this workshop, Francesca Finoccharo (eProsima) presented in detail the micro-ROS architecture, supported platforms and reported recent developments and features: A versatile API for custom transports, continuous fragment mode, static memory pools in the RMW and time synchronization between client and agent. ([slides](/download/2021-04-13-ERF-eprosima.pdf))
```

File: _posts/2021-04-13-ERF2021.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Then, Jan Staschulat (Bosch) gave an overview of the RCLC API of micro-ROS and highlighted the deterministic RCLC Executor. ([slides](/download/2021-04-13-ERF-bosch.pdf)) Two customers of micro-ROS presented their industrial use-cases and emphasized the benefit of micro-ROS: First, Ana&euml;lle Sarazin (WYCA Robotics) presented "Elodie and micro-ROS" ([slides](/download/2021-04-13-ERF-wyca.pdf)) and, secondly, Tomasz Rokosz (Hydra System) presented "micro-ROS enabled GNSS receiver" ([slides](/download/2021-04-13-ERF-hydra.pdf)) ([video of the demo](https://www.youtube.com/watch?v=hTEnTju1Qkg))
```
Comment (pablogs9) [inline _posts/2021-04-13-ERF2021.md:12]:
```suggestion
Then, Jan Staschulat (Bosch) gave an overview of the RCLC API of micro-ROS and highlighted the deterministic RCLC Executor. ([slides](/download/2021-04-13-ERF-bosch.pdf)) Two customers of micro-ROS presented their industrial use-cases and emphasized the benefit of micro-ROS: First, Analle Sarazin (WYCA Robotics) presented "Elodie and micro-ROS" ([slides](/download/2021-04-13-ERF-wyca.pdf)) and, secondly, Tomasz Rokosz (Hydra System) presented "micro-ROS enabled GNSS receiver" ([slides](/download/2021-04-13-ERF-hydra.pdf)) ([video of the demo](https://www.youtube.com/watch?v=hTEnTju1Qkg))
```

File: _posts/2021-04-13-ERF2021.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Finally, we hosted a panel discussion with Pablo Garrido Sanchez (eProsima), Alexandre Malki (PIAP), Jan Staschulat (Bosch), Ana&euml;lle Sarazin (WYCA Robotics) and Tomasz Rokosz (Hydra System) which stimulated a very lively participation of the audience.
```
Comment (pablogs9) [inline _posts/2021-04-13-ERF2021.md:14]:
```suggestion
Finally, we hosted a panel discussion with Pablo Garrido Sanchez (eProsima), Alexandre Malki (PIAP), Jan Staschulat (Bosch), Anaelle Sarazin (WYCA Robotics) and Tomasz Rokosz (Hydra System) which stimulated a very lively participation of the audience.
```

**Review by pablogs9 — APPROVED** [review]

**Comment by pablogs9** [comment]
I have just updated the URL to a more representative FreeRTOS + POSIX webpage

**Review by pablogs9 — DISMISSED** [review]

File: _posts/2021-04-13-ERF2021.md (hunk 1 above)
Comment (ralph-lange) [reply _posts/2021-04-13-ERF2021.md:10]: You may even remove the "In this workshop" now, given may proposed addition to the very first sentence.

**Review by ralph-lange — CHANGES_REQUESTED** [review]
Please embed the slides and videos directly in the post:
* Slides can be embedded with `<embed src="/download/thefilename.pdf" type="application/pdf" width="100%" height="450px"/>`, see https://github.com/micro-ROS/micro-ROS.github.io/blame/master/_posts/2020-03-03-micro-ROS-at-ERF2020.md#L16.
* The Share button at YouTube allows getting the HTML snippet to embed the video.

**Comment by JanStaschulat** [comment]
> Please embed the slides and videos directly in the post:
> 
>     * Slides can be embedded with `<embed src="/download/thefilename.pdf" type="application/pdf" width="100%" height="450px"/>`, see https://github.com/micro-ROS/micro-ROS.github.io/blame/master/_posts/2020-03-03-micro-ROS-at-ERF2020.md#L16.
> 
>     * The Share button at YouTube allows getting the HTML snippet to embed the video.

thanks for the feedback. It is updated in commit https://github.com/micro-ROS/micro-ROS.github.io/pull/318/commits/48d58df17f11fb310c1353ed7475e6fd8101e632

File: _posts/2021-04-13-ERF2021.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Two customers of micro-ROS presented their industrial use-cases and emphasized the benefit of micro-ROS: First, Ana&euml;lle Sarazin (WYCA Robotics) presented "Elodie and micro-ROS":
+
+<embed src="/download/2021-04-13-ERF-wyca.pdf" type="application/pdf" width="100%" height="450px"/>
+
+and, secondly, Tomasz Rokosz (Hydra System) presented "micro-ROS enabled GNSS receiver":
+
+<embed src="/download/2021-04-13-ERF-hydra.pdf" type="application/pdf" width="100%" height="450px"/>
+
+Video of the demo by Hydra Systems:
```
Comment (ralph-lange) [inline _posts/2021-04-13-ERF2021.md:22]:
```suggestion
Video of the demo by Hydra System:
```

File: _posts/2021-04-13-ERF2021.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Then, Jan Staschulat (Bosch) gave an overview of the RCLC API with a code example and highlighted the deterministic behavior of the RCLC Executor: 
+
+<embed src="/download/2021-04-13-ERF-bosch.pdf" type="application/pdf" width="100%" height="450px"/>
+
+Two customers of micro-ROS presented their industrial use-cases and emphasized the benefit of micro-ROS: First, Ana&euml;lle Sarazin (WYCA Robotics) presented "Elodie and micro-ROS":
+
+<embed src="/download/2021-04-13-ERF-wyca.pdf" type="application/pdf" width="100%" height="450px"/>
+
+and, secondly, Tomasz Rokosz (Hydra System) presented "micro-ROS enabled GNSS receiver":
```
Comment (ralph-lange) [inline _posts/2021-04-13-ERF2021.md:18]:
```suggestion
And, secondly, Tomasz Rokosz (Hydra System) presented "micro-ROS enabled GNSS receiver":
```

**Review by ralph-lange — APPROVED** [review]

**Comment by JanStaschulat** [comment]
I updated the blog post accordingly. However, I am not authorized to merge this pull request. @pablogs9 or @ralph-lange, could you merge it?

**Pull request merged**